spec/integration/runners/queue/rspec_runner_spec.rb

```diff
@@ -1403,12 +1403,17 @@ def when_first_matching_example_defined(type:)
         OUTPUT
       )
 
+      expect(actual.stdout).to include('To retry the last batch of tests fetched from the Queue API, please run the following command on your machine:')
+      expect(actual.stdout).to include('bundle exec rspec --format documentation --default-path spec_integration "spec_integration/b_spec.rb" "spec_integration/c_spec.rb"')
 
       expect(actual.stdout).to include('Use the following backtrace(s) to find the line of code that got stuck if the CI node hung and terminated your tests.')
+      expect(actual.stdout).to include('Hanging specs in the main thread:')
+      expect(actual.stdout).to include('Hanging specs in non-main thread:')
       expect(actual.stdout).to include('Main thread backtrace:')
-      expect(actual.stdout).to match(/spec_integration\/b_spec\.rb:7:in .*kill/)
+      expect(actual.stdout.scan(/spec_integration\/b_spec\.rb:7:in .*kill/).size).to eq 2
```

Comment:
recommendation

This reads a bit better:
```suggestion
      expect(actual.stdout).to match(/spec_integration\/b_spec\.rb:7:in .*kill/).twice
```